commitments/mode.go

```diff
@@ -4,10 +4,22 @@ import (
 	"fmt"
 )
 
+// EncodingType represents the serialization format for the certificate
+type EncodingType byte
+
+const (
+	// RLPEncoding represents RLP encoding (default)
+	RLPEncoding EncodingType = iota
+	// ABIVerifyV2CertEncoding represents ABI encoding
+	ABIVerifyV2CertEncoding
+)
+
```

Comment:
oh I see you redefine it here... cant the client just import this? and we can have a translation between byte and string if needed

Reply:
that'd be nice but don't think its feasible given how the workspace is configured

Reply:
see https://github.com/Layr-Labs/eigenda-proxy/pull/376 for an example